cetloxblock/docs/TipsAndTricks.html

```diff
@@ -0,0 +1,106 @@
+  </ul>
+
+  <h1>campus</h1>
+  <ul>
+    <li>LMS - <a target="_blank" href="https://campus-dev.opencraft.hosting/">https://campus-dev.opencraft.hosting/ </a></li>
+    <li>CMS - <a target="_blank" href="https://studio-campus-dev.opencraft.hosting/">https://studio-campus-dev.opencraft.hosting/ </a></li>
+  </ul>
+  <h2>prod IDM</h2>
+  <ul>
+    <li>משתמש  : 4125482 </li>
+    <li>סיסמה : My123456</li>
+  </ul>
+
+  <h2>testing IDM</h2>
+  <ul>
+    <li>משתמש: 1352200</li>
+    <li>סיסמה: Yz123456</li>
+  </ul><ul>
+    <li>משתמש: 4125482</li>
+    <li>סיסמה: My123456</li>
+  </ul><ul>
+  <li>משתמש: 1352198</li>
+  <li>סיסמה: Yz123456</li>
+  <li> (משויך ל2 בתי ספר פעם כמורה פעם כתלמיד)</li>
+</ul>
+  <h1>דוגמה לדוחות אדפטיביים</h1>
+  <a target="_blank" href="http://dashboard.lab.cet.ac.il//ability/dashboard-data/c2e7eb3e-4a22-4bd0-bb6f-8f5405884d9e/4c9a96db-cec0-4af3-b8d2-ea61a952d12b">http://dashboard.lab.cet.ac.il//ability/dashboard-data/c2e7eb3e-4a22-4bd0-bb6f-8f5405884d9e/4c9a96db-cec0-4af3-b8d2-ea61a952d12b  </a><br />
+  מתוך <a target="_blank" href="http://productplayer.lab.cet.ac.il/fractions2018/abec9d8d-bba9-4ca3-91ae-d44ae37b2c5b">http://productplayer.lab.cet.ac.il/fractions2018/abec9d8d-bba9-4ca3-91ae-d44ae37b2c5b (עומרמורה) </a><br />
+  הקוד בתוך הפרויקט dashboard <br />
+
+  <h1>Xblock development undocumented stuff</h1>
+  <ol>
+    <li>To update an installed xblock – upgrade the version number in setup.py and reinstall.</li>
+    <li>Xblock.author_view is used by studio only if xblock.has_author_view = True .</li>
+    <li>The editor's save/cancel buttons are part of the xblock's html! Including their binding to click and to studio_submit handler. (a good examle: simplevideo  <a target="_blank" href="http://opencraft.com/doc/edx/xblock/tutorial.html">http://opencraft.com/doc/edx/xblock/tutorial.html) </a></li>
+    <li>If missing, studio displays a 'close' button that only closes the editor.</li>
+    <li>See simplevideo tutrial for runtime.notify('save'). It triggers 'saving…' label and a reloads author view when done.</li>
+    <li>The xblock title in the studio is set by the field 'display_name' (scope.settings).</li>
+  </ol>
+
+  <h1>run xblock</h1>
+  <h2>To run devstack</h2>
+  vagrant up <br />
+  vagrant ssh <br />
+  sudo su edxapp <br />
+  paver devstack studio [--fast] <br />
+
+  <h3>to close</h3>
+  ^C <br />
+  ^D <br />
+  ^D <br />
+  vagrant halt <br />
+
+  <h2>to update xblock</h2>
+  When studio is running, open a new console <br />
+  vagrant ssh <br />
+  sudo -u edxapp /edx/bin/pip.edxapp install --upgrade --no-deps  /vagrant/edx-platform/xblocks/cetloxblock <br />
+
+  <h2>To run xblock-sdk env</h2>
+  D: <br />
+  cd \xblockEdxDev <br />
+  workon testenv <br />
+  cd xblock-sdk <br />
+  python manage.py runserver <br />
+
+  <h3>to close</h3>
+  ^C <br />
+  deactivate <br />
+
+</body>
+</html>
```

Comment:
Nit: needs extra newline.

Reply:
This whole file was not supposed to be a part of the component and will be deleted.